src/chart/layout/addOverlay.js

```diff
@@ -0,0 +1,8 @@
+export default function addOverlay() {
+    this.svg
```

Comment:
worth saving the object as `this.overlay`? Any reason not to?

Reply:
I don't see why not, especially since I _think_ we use the overlay when we're setting up brushable charts.